README.md

```diff
@@ -192,10 +192,24 @@ Dependency-Track based on the metadata present in the BOM:
 * This requires a CycloneDX BOM using Schema 1.2 or later.
 * Not all information is visible in the Dependency-Track server UI.
 
-| Property          | Required  | Default Value  |
-|-------------------|-----------|----------------|
-| bomLocation       | false     | target/bom.xml |
-| updateProjectInfo | false     | false          |
+From Dependency-Track server 4.8.0 onwards, you can set the project parent by setting `updateParent` to `true`. The 
+parent name will be defaulted to that POM's project parent name. If you wish to override that value, or there is 
+no parent set within the `pom.xml`, then explicitly set `parentName` and `parentVersion`.  
+
+Dependency Track doesn't require a project version to be set which means this will fail if there is no version set on 
+the project in Dependency Track that matches the name of the parent of the maven project or the overridden parentName 
+value.
+
+**Note:** If the parent cannot be found on the Dependency-Track server, the BOM upload will not be attempted in order to
+prevent a project being incorrectly created or updated the server.
+
+| Property          | Required | Default Value             | Example Values        |
+|-------------------|----------|---------------------------|-----------------------|
+| bomLocation       | false    | target/bom.xml            | target/custom-bom.xml |
+| updateProjectInfo | false    | false                     | false                 |
+| updateParent      | false    | false                     | true                  |
+| parentName        | false    | ${project.parent.name}    | my-name-override      |
+| parentVersion     | false    | ${project.parent.version} | my-version-override   |
```

Comment:
@roadSurfer I've made some behaviour tweaks from your original PR best documented here.

In my view not defaulting the parent version to that of the maven parent pom seemed counter-intuitive.  I see where you are coming from in that DT doesn't mandate a version on a project.  However if that project was created by the plugin, then the version will be set.  If the project wasn't created by this plugin and is missing a version, it can be added in the DT UI and then set as the `parentVersion` property in the use of this plugin to link it up.

Does that sound reasonable to you and not too far from your original intent?

Reply:
Seems totally fine to me.